File: _archive/Elements/sandbox/src/de/felixroske/sandbox/MyClass.java
package de.felixroske.sandbox;

public class MyClass {

    public static void main(String[] args) {

        int currentInt;

        MyClass myClass = new MyClass();
        
        
        myClass.doSomething(Integer.valueOf("42"));
    }

    public Integer doSomething(int x) {

        switch (x) {
        case 1:
            
            
        }
        return Integer.valueOf(x);
    }
}

File: Elements/BasisElementeSW/hello-world/flx-kotlin-project/src/MyClass.kt
package de.felixroske.kotlin

import java.util.UUID

data class MyClass(val name: String? = null, val id: UUID) {
}

fun main() {

    var mytest: MyClass? = MyClass(
        id = UUID.randomUUID(),
        name = "TestName"
    )

    println("Der Name ist ${mytest?.name}")

    mytest?.let {
        println(it.name)
    }

    doSomething(mytest)?.let {
        println(it)
    }
}

class AnotherClass(var name: String)

fun doSomething(myclass: MyClass?): String? {

    val anotherClass = AnotherClass("Hello")

    print(myclass!!.name)
    anotherClass.name = "AnotherName"

    return anotherClass.name
}
